src/FastRandom.h

```diff
@@ -97,8 +97,8 @@ class cRandomWrapper
 
 
 	/** Return a random IntType in the range [0, a_Max]. */
-	template <class IntType = int, class ArgType>
-	IntType RandInt(ArgType a_Max)
+	template <class IntType = int>
+	IntType RandInt(IntType a_Max)
```

Comment:
Shouldn't the `RandInt(a_Min, a_Max)` overload be changed as well as the `RandReal` versions or they will have different behaviour.